packages/postgres-database/src/simcore_postgres_database/models/funcapi_functions_table.py

```diff
@@ -0,0 +1,80 @@
+"""Functions table
+
+- List of functions served by the simcore platform
+"""
+
+import uuid
+
+import sqlalchemy as sa
+from simcore_postgres_database.models._common import (
+    column_created_datetime,
+    column_modified_datetime,
+)
+from sqlalchemy.dialects.postgresql import UUID
+
+from .base import metadata
+
+functions_table = sa.Table(
+    "funcapi_functions",
+    metadata,
+    sa.Column(
+        "uuid",
+        UUID(as_uuid=True),
+        primary_key=True,
+        index=True,
+        default=uuid.uuid4,
+        doc="Unique id of the function",
+    ),
+    sa.Column(
+        "title",
+        sa.String,
+        doc="Name of the function",
+    ),
+    sa.Column(
+        "function_class",
+        sa.String,
+        doc="Class of the function",
+    ),
+    sa.Column(
+        "description",
+        sa.String,
+        doc="Description of the function",
+    ),
+    sa.Column(
+        "input_schema",
+        sa.JSON,
+        doc="Input schema of the function",
+    ),
+    sa.Column(
+        "output_schema",
+        sa.JSON,
```

Comment:
same for these json ones

Reply:
Ok, it doesn't seem to be consistent in the other tables, but I'll switch to JSONB